tests/stress_tests/common/tests_utils.cpp

```diff
@@ -8,6 +8,14 @@
 #include <pugixml.hpp>
 #include <string>
 
+#ifdef _WIN32
```

Comment:
are such includes really required? I can't find any use of it here

Reply:
Oh thanks, you're right 👍